recipes/approvaltests.cpp/7.0.0/test_package/conanfile.py

```diff
@@ -0,0 +1,20 @@
+import os
+from conans import ConanFile, CMake, tools
+
+
+class TestPackageConan(ConanFile):
+    settings = "os", "compiler", "arch", "build_type"
+    generators = "cmake"
```

Comment:
```suggestion
    generators = "cmake"
    options = {"test_framework:catch2": True}
```
Add catch2 as option to install Catch2 for test package.

Reply:
@uilianries If I pull this in I get the following error:

```
approvaltests_cpp/7.0.0: Using the exported files summary hash as the recipe revision: dbd60f04afc29d3d9d9d311388f94ebe 
approvaltests_cpp/7.0.0: Exported revision: dbd60f04afc29d3d9d9d311388f94ebe
Configuration:
[settings]
arch=x86_64
arch_build=x86_64
build_type=Debug
compiler=gcc
compiler.libcxx=libstdc++
compiler.version=8.3
os=Linux
os_build=Linux
[options]
[build_requires]
[env]

ERROR: : 'approvaltests_cpp:test_framework' value not defined
```

Reply:
sorry, forget about, catch2 is already the default one.